### PR TITLE
chore(core): removed obsolete TODO

### PR DESCRIPTION
**Pull request by JimmFly**
Cleaned up some TODOs that no longer need to be done

### CODE REVIEW

**Comment by JimmFly** [comment]
* **#7337** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7337?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JimmFly and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/pages/workspace/detail-page/detail-page.css.ts [hunk 1]
```diff
@@ -22,7 +22,6 @@ export const affineDocViewport = style({
   flexDirection: 'column',
   userSelect: 'none',
   containerName: 'viewport',
-  // TODO(@JimmFly): find out what this does in bs
```
Comment (forehalo) [inline packages/frontend/core/src/pages/workspace/detail-page/detail-page.css.ts:25]: so, found out?

**Review by CatsJuice — APPROVED** [review]

File: packages/frontend/core/src/pages/workspace/detail-page/detail-page.css.ts (hunk 1 above)
Comment (JimmFly) [reply packages/frontend/core/src/pages/workspace/detail-page/detail-page.css.ts:25]: This style itself is copied from [block-suite-editor/styles.css.ts](https://github.com/toeverything/AFFiNE/commit/f832ada98cf8fcf09339bd3a2c08cb95c95f5016#diff-30ec7d1597a28e385440da4a95099eb4ae0dc7a828c2b0bd246cfbb41f642872), and the style inside is completely different.
Regarding this class name, there are many places in blocksuite to query it. This is a required class name.

Comment (pengx17) [reply packages/frontend/core/src/pages/workspace/detail-page/detail-page.css.ts:25]: It is to determine the main doc padding width on the left & right.

**Review by forehalo — APPROVED** [review]

**Pull request merged**